user/iplib/epoll.c

```diff
@@ -158,15 +158,19 @@ static struct event_queue *ep_get_alarm_evq(void)
  * some sort of user deferred destruction. (TODO). */
 static void ep_put_ceq_evq(struct event_queue *ceq_evq)
 {
+#if 0 /* TODO: EVQ/INDIR Cleanup */
 	ceq_cleanup(&ceq_evq->ev_mbox->ceq);
 	evq_remove_wakeup_ctlr(ceq_evq);
 	put_eventq_raw(ceq_evq);
+#endif
```

Comment:
Is the change here that the race triggered this code to break so you just commented it out until you come up with a proper fix?  If it's benign to not do the cleanup, then I think your temporary fix is fine, otherwise, maybe print a warning so we know that code went down this path rather than just silently do nothing...